app/src/main/res/values-de/strings.xml

```diff
@@ -1,5 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <string name="abc_action_bar_home_description">Zur Startseite</string>
```

Comment:
The name start with "abc" was can't be translated, I already remove it
> I had merge your pull request, and removed some strings which can't be translated.
All the app strings you can see at /app/src/main/res/values/strings.xml, some strings marked "translatable="false"", meaning it can't be translated. If there are any new strings needed in app, I will add it at the end of file

You should improve translation base on [string.xml](https://github.com/ThirtyDegreesRay/OpenHub/blob/master/app/src/main/res/values-de/strings.xml), I am sorry about that I did't say it clearly.